apply-patch.sh

```diff
@@ -48,6 +48,7 @@ ORA_DB_NAME_PARAM="^[a-zA-Z0-9_$]+$"
 #
 INVENTORY_FILE="${INVENTORY_FILE:-./inventory_files/inventory}"
 
+export ANSIBLE_DISPLAY_SKIPPED_HOSTS=false
```

Comment:
Can we add comment something like (from the docs): 
```
# Override default Ansible behavior and avoid displaying skipped task/host entries in a task 
```

Reply:
Done.